doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/configs/xgboost-benchmark.yaml

```diff
@@ -0,0 +1,48 @@
+# This is a Ray cluster configuration for exploration of the 100Gi XGBoost-Ray workload.
```

Comment:
nit:
`XGBoost-Ray workload` -> `Ray AIR XGBoostTrainer benchmark`

We should probably modify the k8s example yaml too.